### PR TITLE
remove keyword object creation

### PR DESCRIPTION
**Pull request by perryr16**
## Fear Quiz Bugs
* remove fears that produced no results
* added a 404-error fear

## Remove keyword object generation
* slowing results down
* just commented out

## Keyword Search
* if there is a space, replace with %20

### CODE REVIEW

File: app/controllers/game_search_controller.rb [hunk 1]
```diff
@@ -27,7 +27,9 @@ def create_game_objects(params)
     if params[:search_type] == 'game'
       game = results.create_game_objects(params[:search])
     elsif params[:search_type] == 'keyword'
-      games = results.games_by_keywords(params[:search])
+      keyword = params[:search].gsub(" ", "%20")
```
Comment (perryr16) [inline app/controllers/game_search_controller.rb:30]: hoping this fixes 'tom nook'. worked locally but we'll see on heroku

File: spec/features/quiz/quiz_results_spec.rb [hunk 2]
```diff
@@ -83,10 +83,10 @@
     game1 = Game.first
     game2 = Game.last
 
-    game1_keywords = game1.keywords.map(&:keyword)
-    expect((game1_keywords & selected_keywords).present?).to eq(true)
-    game2_keywords = game2.keywords.map(&:keyword)
-    expect((game2_keywords & selected_keywords).present?).to eq(true)
+    # game1_keywords = game1.keywords.map(&:keyword)
```
Comment (perryr16) [inline spec/features/quiz/quiz_results_spec.rb:86]: not testing keywords

File: app/views/fear_quiz/new.html.erb [hunk 3]
```diff
@@ -51,22 +51,22 @@
           <td>
             <%= check_box_tag 'box31', 'outdoors' %> outdoors <br>
             <%= check_box_tag 'box32', 'parents' %> parents <br>
-            <%= check_box_tag 'box33', 'rabbits' %> rabbits <br>
+            <%= check_box_tag 'error_404', '404' %> 404 errors <br>
```
Comment (perryr16) [inline app/views/fear_quiz/new.html.erb:54]: if a user checks this box and clicks enter it will redirect to a 404. stupid? yes.

Comment (whitneykidd) [reply app/views/fear_quiz/new.html.erb:54]: this is dumb. i love it.

**Pull request merged**